src/main/java/com/teragrep/rlp_03/context/ConnectionContextImpl.java

```diff
@@ -123,6 +123,7 @@ public void close() {
 
     @Override
     public void handleEvent(SelectionKey selectionKey) {
+        // TODO refactor this into Strategy pattern
```

Comment:
TODO: Create ticket if important, otherwise leave as a note for future development